config/connectors.yml.example

```diff
@@ -19,6 +19,7 @@ thread_pool:
   max_queue: 100
 
 log_level: info
+ecs_logging: true
```

Comment:
I try to make it 
```
ecs_logging: ${ECS_LOGGING:false}
```

But it fails validation when `ECS_LOGGING` is not set:
```
ecs_logging: must be boolean
```

I have no idea why it doesn't work (when `ECS_LOGGING` is set to `ture` or `false`, it works fine), and I will leave it true by default.